index.js

```diff
@@ -341,6 +342,26 @@ var Purchases = /** @class */ (function () {
         return RNPurchases.createAlias(newAppUserID);
     };
     /**
+     * This function will logIn the current user with an appUserID. Typically this would be used after a logout to identify a new user without calling configure
+     * @param {String} newAppUserID The appUserID that should be linked to the currently user
```

Comment:
this won't always be a new user though, right? they could actually be logging out and back in as different, already-created users to an app? just wondering if the naming in the comment could be more generic

Reply:
That's actually a good catch. The function parameter is `appUserID` so this should be `appUserID` otherwise I don't think the JSDoc will work as intended

Reply:
yeah, that's a bad copy/paste on my part. thanks for catching!